package.json

```diff
@@ -188,7 +188,8 @@
     "resolutions": {
         "@nx/js@16.3.2": "patch:@nx/js@npm%3A16.3.2#./.yarn/patches/@nx-js-npm-16.3.2-a543eca792.patch",
         "postcss": "^8.4.31",
-        "word-wrap": "^1.2.4"
+        "word-wrap": "^1.2.4",
+        "@sideway/formula": "^3.0.1"
```

Comment:
is this because some dependency depends on an older version of this?

Reply:
yea same thing to try and get it to apply a patch change to this one: 
<img width="585" alt="image" src="https://github.com/DevCycleHQ/js-sdks/assets/1219069/f75ef6e5-7bbf-47c7-94ab-b9adde3d6b63">